doc/link.md

```diff
@@ -0,0 +1,58 @@
+@function can-route.link link
+@parent can-route.static
```

Comment:
move into a `deprecated` group.

Reply:
I added deprecate .link() to #209 and put a deprecation warning on the page. Even adding a group for deprecated/can-route.deprecated and setting the group as can-route.link's parent didn't seem to work.